Source/WebKit/GPUProcess/graphics/WebGPU/RemoteDevice.cpp

```diff
@@ -143,7 +145,9 @@ void RemoteDevice::createTexture(const WebGPU::TextureDescriptor& descriptor, We
         return;
 
     auto texture = m_backing->createTexture(*convertedDescriptor);
-    auto remoteTexture = RemoteTexture::create(texture, m_objectHeap, m_streamConnection.copyRef(), identifier);
+    if (!texture)
```

Comment:
Since we don't signal back to the web process, I think it is incorrect to not add the remote texture to the objectHeap, since the web process will be left with an incomplete connection.

Instead, can we have `RemoteTexture::create` return a `Ref` instead of a `RefPtr` please?

Reply:
Using the interface WebCore::WebGpu::Device for both RemoteDeviceProxy where it can fail and DeviceImpl where it can't won't work then.  This null check currently can't be hit